installer/main.sh

```diff
@@ -68,27 +83,29 @@ cd -
 
 
 if [ ! -z "$DESIRED_KOPS_CLUSTER_NAME" ] && [ -z "$SKIP_K8S_INSTALLATION" ]; then
-cd k8s
-. create_kops_cluster_gke.sh $DESIRED_KOPS_CLUSTER_NAME $GCE_ZONE
-. install_tiller.sh
-cd ..
+    cd k8s
+    . create_kops_cluster_gke.sh $DESIRED_KOPS_CLUSTER_NAME $GCE_ZONE
+    . install_tiller.sh
+    cd ..
 fi
 
 cd ingress
 . install.sh
 cd ..
 
 cd crd
-. install.sh $DNS_DOMAIN_NAME
+. install.sh $DNS_DOMAIN_NAME $USE_HTTPS
 cd ..
 
 cd dns
 . setup.sh $DNS_DOMAIN_NAME
 cd ..
 
-cd minio
-. install.sh $MINIO_ACCESS_KEY $MINIO_SECRET_KEY $MINIO_URL
-cd ..
+if $INSTALL_MINIO ; then
```

Comment:
i'd change that to if [[ "$INSTALL_MINIO" ]]; then
(to make it consistent)

(or even [[ -n "$INSTALL_MINIO" ]])